crates/primitives/src/chain.rs

```diff
@@ -118,6 +124,46 @@ impl Chain {
         }
     }
 
+    pub fn hrp(&self) -> &str {
```

Comment:
There is `CosmosChain` type, maybe have `hrp` under it?

Reply:
good point